themes/custom_theme/valencia-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,565 @@
+		box-shadow: none;
+		> a:hover {
+			color: $accent-color;
+		}
+		& h1 {
+			color: #fff;
+		}
+		@media #{$screen-sm} {
+			background-color: transparent;
+		}
+	}
+    .navbar-nav {
+        > li {
+			padding: 0 10px;
+			margin-right: 10px;
+			margin-left: 10px;
+			@media #{$screen-sm} {
+				display: inline-block;
+			}
+			@media #{$screen-md} {
+				margin: 0;
+				&.btn {
+					margin: 0 10px;
+					&:last-child {
+						margin-right: 0;
+					}
+				}
+			}
+            > a {
+                padding-top: 15px;
+                padding-bottom: 15px;
+                border-radius: 3px;
+                color: inherit;
+                font-size: 12px;
+                font-weight: 400;
+                text-transform: uppercase;
+            }
+            > a:hover, a:focus {
+                color: $accent-color;
+            }
+        }
+    }
+	.title-logo-wrapper {
+    min-width: 250px;
+	}
+}
+
+.navbar-default .navbar-toggle {
+	border: none;
+}
+
+.navbar-default .navbar-toggle:hover,
+.navbar-default .navbar-toggle:focus {
+	background-color: transparent;
+}
+
+.navbar .navbar-toggle .icon-bar {
+	border: #fff 1px solid;
+	background-color: inherit;
+}
+
+.navbar-default:not(.navbar-transparent) .navbar-nav > li > a.btn.btn-white.btn-simple {
+	color: #555;
+}
+
+.navbar.navbar-color-on-scroll li.active a {
+	color: #9c27b0;
+}
+
+.navbar.navbar-color-on-scroll li.active.btn a,
+.navbar.navbar-transparent li.active a {
+	color: #fff;
+}
+
+.navbar.navbar-transparent ul.dropdown-menu li a {
+	color: #333;
+}
+
+@media #{$screen-sm} {
+
+	.dropdown .dropdown-menu {
+		display: block;
+		visibility: hidden;
+		margin-top: -20px;
+		opacity: 0;
+		transition: all 150ms linear;
+	}
+
+	.dropdown.open .dropdown-menu {
+		visibility: visible;
+		margin-top: 1px;
+		opacity: 1;
+	}
+
+    .navbar-nav > li {
+        display: inline-block;
+    }
+
+    .navbar .navbar-collapse {
+        display: table-cell !important;
+        vertical-align: middle;
+        text-align: right;
+        width: 100%;
+    }
+}
+
+/* ============================================================
+    Typography
+=============================================================== */
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    line-height: 1.5em;
+}
+
+h1,
+h3 {
+    font: {
+        weight: 700;
+        family: $font-serif;
+    }
+    color: $font-h-color;
+}
+
+h2,
+h4,
+h6 {
+    font: {
+        weight: 700;
+        family: $font-sans;
+    }
+    color: $font-h-color;
+}
+
+h1 {
+	font-size: 3.8em;
+	line-height: 1.15em;
+}
+
+h2 {
+	font-size: 2.6em;
+}
+
+h3 {
+	font-size: 1.825em;
+}
+
+h4 {
+	font-size: 1.25em;
+	line-height: 1.55em;
+}
+
+h5 {
+	font-size: 1.25em;
+	line-height: 1.55em;
+    font-family: $font-sans;
+    color: #777;
+    font-weight: 300;
+}
+
+h6 {
+	font-size: 0.9em;
+	font-weight: 500;
+	text-transform: uppercase;
+}
+
+p {
+    margin-bottom: 10px;
+    font-size: 1.142em;
+    color: $font-p-color;
+}
+
+.white {
```

Comment:
We should use a more semantic CSS class since color's or positions are not very flexible in case we decide to change the color later on.